Composer/packages/client/src/components/GetStarted/GetStartedToolbar.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+const items: IToolbarItem[] = [
```

Comment:
convert this to a function so those formatMessages get executed in the context of a component for them to get picked up by localization.

getToolbarItems = () => items

Reply:
this file was removed